roles/edpm_bootstrap/tasks/bootstrap.yml

```diff
@@ -160,3 +160,47 @@
       ansible.builtin.service:
         name: NetworkManager
         state: reloaded
+
+- name: Configure swap file
```

Comment:
According to the original [tripleo design spec](https://opendev.org/openstack/tripleo-specs/commit/59e352f1a928c8469cb46756eac6113397d06de4) which established the standalone roles aka EDPM for NG, there are the following tasks (stages):

* bootstrap
* install
* pre-network
* network
* configure
* container-config
* service-bootstrap

While those are likely a subject of changes, the idea behind, I believe is to reduce the need for future migrations, like using mutating hooks to update all CRs for EDPM objects, when changing the tasks and roles (if we expose it as a public/partners integration interface)

So, no, we should keep it in the bootstrap.yaml tasks

Reply:
I don't have any strong opinion on splitting it out into a separate role. A separate task file would be fine though that could just be included here. It might make things a bit cleaner, and adds the ability to execute just the swap tasks if needed.